networks/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(scitile/LAS)
```

Comment:
Add a newline to the end of this file. (The red minus in a circle GitHub added is indicating that no terminal newline is present.)